setup.cfg

```diff
@@ -66,3 +66,6 @@ max-complexity = 15
 select = B,C,E,F,W,T4,B902,T,P
 show_source = true
 count = true
+
+[bandit]
```

Comment:
`bandit` isn't an scmrepo dependency and should not be defined here. The issue is that it's used in the pre-commit virtualenv as a part of the flake8 hook, so this fix needs to go in `.pre-commit-config.yaml` like:

```
  - hooks:
      - id: flake8
        additional_dependencies:
          ...
          - flake8-bandit<1.7.3
```